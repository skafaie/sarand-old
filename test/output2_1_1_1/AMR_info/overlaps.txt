SHV-52:SHV-24, SHV-41, SHV-42, SHV-18, SHV-79, SHV-105, SHV-12, SHV-64, SHV-182, SHV-92, SHV-56, SHV-89, SHV-61, SHV-101, SHV-35, SHV-121, SHV-137, SHV-152, SHV-164, SHV-128, SHV-162, SHV-180, SHV-32, SHV-78, SHV-188, SHV-6, SHV-15, SHV-8, SHV-21, SHV-36, SHV-69, SHV-83, SHV-67, SHV-133, SHV-49, SHV-156, SHV-38, SHV-185, SHV-9, SHV-82, SHV-11, SHV-100, SHV-75, SHV-153, SHV-30, SHV-142, SHV-108, SHV-37, SHV-129, SHV-62, SHV-72, SHV-97, SHV-172, SHV-102, SHV-43, SHV-57, SHV-46, SHV-14, SHV-150, SHV-141, SHV-20, SHV-157, SHV-161, SHV-124, SHV-33, SHV-53, SHV-122, SHV-106, SHV-186, SHV-96, SHV-93, SHV-66, SHV-77, SHV-134, SHV-25, SHV-74, SHV-1, SHV-76, SHV-2, SHV-103, SHV-71, SHV-158, SHV-59, SHV-109, SHV-22, SHV-55, SHV-168, SHV-173, SHV-119, SHV-81, SHV-31, SHV-149, SHV-34, SHV-147, SHV-65, SHV-126, SHV-94, SHV-167, SHV-60, SHV-125, SHV-19, SHV-80, SHV-26, SHV-63, SHV-13, SHV-135, SHV-99, SHV-48, SHV-107, SHV-85, SHV-51, SHV-143, SHV-104, SHV-140, SHV-187, SHV-110, SHV-123, SHV-2A, SHV-23, SHV-86, SHV-45, SHV-154, SHV-84, SHV-70, SHV-95, SHV-40, SHV-111, SHV-73, SHV-151, SHV-16, SHV-28, SHV-148, SHV-159, SHV-183, SHV-120, SHV-5, SHV-127, SHV-27, SHV-29, SHV-160, SHV-98, SHV-50, SHV-155, SHV-189, SHV-44, SHV-144, SHV-7, SHV-145, SHV-178, SHV-165, SHV-163, SHV-179, SHV-3, SHV-4
TEM-126:TEM-72, TEM-59, TEM-34, TEM-7, TEM-132, TEM-47, TEM-16, TEM-102, TEM-116, TEM-109, TEM-43, TEM-87, TEM-183, TEM-198, TEM-195, TEM-78, TEM-121, TEM-138, TEM-206, TEM-92, TEM-158, TEM-91, TEM-11, TEM-67, TEM-17, TEM-207, TEM-176, TEM-148, TEM-166, TEM-1, TEM-112, TEM-155, TEM-122, TEM-214, TEM-71, TEM-33, TEM-2, TEM-182, TEM-22, TEM-151, TEM-128, TEM-191, TEM-188, TEM-84, TEM-52, TEM-95, TEM-133, TEM-106, TEM-107, TEM-187, TEM-110, TEM-73, TEM-145, TEM-201, TEM-21, TEM-89, TEM-162, TEM-10, TEM-120, TEM-159, TEM-205, TEM-213, TEM-192, TEM-30, TEM-217, TEM-6, TEM-55, TEM-171, TEM-26, TEM-113, TEM-83, TEM-68, TEM-163, TEM-142, TEM-42, TEM-185, TEM-63, TEM-49, TEM-152, TEM-125, TEM-88, TEM-137, TEM-96, TEM-153, TEM-209, TEM-29, TEM-28, TEM-136, TEM-118, TEM-20, TEM-169, TEM-79, TEM-186, TEM-90, TEM-156, TEM-146, TEM-129, TEM-54, TEM-134, TEM-219, TEM-178, TEM-144, TEM-114, TEM-45, TEM-104, TEM-141, TEM-177, TEM-93, TEM-149, TEM-193, TEM-124, TEM-85, TEM-15, TEM-40, TEM-216, TEM-157, TEM-199, TEM-75, TEM-160, TEM-168, TEM-130, TEM-81, TEM-194, TEM-197, TEM-139, TEM-117, TEM-80, TEM-211, TEM-143, TEM-135, TEM-115, TEM-184, TEM-101, TEM-53, TEM-127, TEM-19, TEM-12, TEM-48, TEM-164, TEM-147, TEM-131, TEM-8, TEM-3, TEM-94, TEM-82, TEM-70, TEM-86, TEM-208, TEM-150, TEM-167, TEM-76, TEM-154, TEM-105, TEM-190, TEM-111, TEM-215, TEM-24, TEM-189, TEM-57, TEM-123, TEM-108, TEM-60, TEM-196, TEM-4, TEM-220
AAC(6;)-Ib:AAC(6;)-Ib-Suzhou, AAC(6;)-Ib-Hangzhou, AAC(6;)-Ib4, AAC(6;)-Ib3, AAC(6;)-Ib;, AAC(6;)-Ib11
aadA23:aadA21, aadA, aadA15, aadA22, aadA24
FosA5:FosA6
aadA12:aadA17, aadA25, aadA2, aadA3, aadA8, aadA8b
OXA-9:
APH(3;)-Ia:
mecR1:
eptA:
catI:plasmid-encoded_cat_(pp-cat)
mecI:
sul2:
ugd:
ErmA:
AAC(6;)-Ib9:AAC(6;)-Ib10, AAC(6;)-Ib-cr
APH(6)-Id:
sul1:
mecA:
ANT(2;;)-Ia:
ANT(9)-Ia:
dfrA14:
tetM:
AAC(6;)-Ie-APH(2;;)-Ia:
bacA:
APH(3;;)-Ib:
Escherichia_coli_ampC1_beta-lactamase:
pmrF:
Escherichia_coli_ampH:
kdpE:
ANT(4;)-Ib:
catII_from_Escherichia_coli_K-12:
Staphylococcus_aureus_FosB:
Escherichia_coli_ampC:
Klebsiella_pneumoniae_OmpK37:
TEM-181:
OmpA:
eptB:
ArnT:
