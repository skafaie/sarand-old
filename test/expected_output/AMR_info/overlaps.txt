ErmA:
ANT(9)-Ia:
AAC(6;)-Ie-APH(2;;)-Ia:
